### PR TITLE
Feature add import api call to the system import licence

### PR DESCRIPTION
**Pull request by jonathangoulding**
https://eaflood.atlassian.net/browse/WATER-4579

As part of the ongoing migration away from the legacy code we have move the logic to import into the system repo.

This is part of a gradual migration and by adding the call to the system we can look to slowly replace the import functionality from the legacy import service into the water-abstraction-system.

### CODE REVIEW

**Comment by jonathangoulding** [comment]
Complete logs for the added licence system queue and handler . 
```
 [08:00:00.033] INFO (1964): licence-import.queue-licences-system: started
| [08:00:10.110] INFO (1964): licence-import.queue-licences-system: finished
[08:00:20.053] INFO (1964): licence-import.import-licence-system: started
[08:00:20.053] INFO (1964): licence-import.import-licence-system: 6/33/03/*S/0033 sent to system import licence. Job number - 1
[08:00:40.049] INFO (1964): licence-import.queue-licences: started
[08:00:40.055] INFO (1964): licence-import.queue-licences - on complete started for 6/33/03/*S/0033
[08:00:40.057] INFO (1964): licence-import.queue-licences: finished
[08:00:50.058] INFO (1964): licence-import.import-licence: started
 numberOfJobs: 72300
[08:00:50.119] INFO (1964): licence-import.import-licence: finished
numberOfJobs: 72300

```

**Review by Cruikshanks — CHANGES_REQUESTED** [review]
Need to sort out the title and description before this is ready for review @jonathangoulding

**Comment by jonathangoulding** [comment]
> Need to sort out the title and description before this is ready for review @jonathangoulding

Updated.

**Comment by Cruikshanks** [comment]
Just recording it here.

The legacy code likes to log 'all the things'. That seems like the right call, but our experience is it actually just becomes noise when trying to dig into an issue. This is especially the case for high-volume things like import licences (80K of them).

Instead, we 'need' to have a log message when something goes wrong. We 'want' to have a message to say importing licences has started, and then when they have finished. We don't 'need' anything in between.

@jonathangoulding has come up with an improved version of how we handled this in [water-abstraction-service](https://github.com/DEFRA/water-abstraction-service) so is going to make those changes and then re-submit for review.

File: src/modules/licence-import/jobs/queue-licences.js [hunk 1]
```diff
@@ -16,36 +15,27 @@ function createMessage () {
...
-  } catch (error) {
-    global.GlobalNotifier.omfg(`${JOB_NAME}: errored`, error)
-    throw error
-  }
 }
 
 async function onComplete (messageQueue, job) {
   if (!job.failed) {
-    const { value: licences } = job.data.response
-    const numberOfJobs = licences.length
```
Comment (jonathangoulding) [inline src/modules/licence-import/jobs/queue-licences.js:34]: After discussion put this back.

**Review by Cruikshanks — APPROVED** [review]

**Pull request merged**